scripts/archivenode.bash

```diff
@@ -5,16 +5,33 @@
 # Archive node script
 # NB:  you can also download archives at quicksync:
 # https://quicksync.io/networks/osmosis.html
-# 2nd NB:  may explode but works for me. - Jacob
+# 2nd NB: you can change OSMOSISD_PRUNING=nothing to OSMOSISD_PRUNING=default OR you could also set the pruning settings manually with OSMOSISD_PRUNING=custom
+# 3rd NB: you might want to use this to test different databases, and to do that my recommended technique is like:
+# go install -ldflags '-w -s -X github.com/cosmos/cosmos-sdk/types.DBBackend=rocksdb' -tags rocksdb ./...
+# if you do not use the ldflags thing you won't use the chosen db for everything, so best use it.
 
 
-git checkout v1.0.1
+export OSMOSISD_PRUNING=nothing
+
+# VERSION THREE
+echo "v3 took" > howlong
+git checkout v3.x
+time osmosisd start 
+make install
```

Comment:
How does this work? The prior code that downloaded the genesis file made sense to me

Reply:
it wouldn't!

thanks @ValarDragon